core/goroutine/goRoutinesProcessor.go

```diff
@@ -0,0 +1,65 @@
+package goroutine
+
+import (
+	"bytes"
+	"strings"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+type goRoutinesProcessor struct {
+}
+
+// NewGoRoutinesProcessor creates a new GoRoutinesProcessor
+func NewGoRoutinesProcessor() core.GoRoutinesProcessor {
```

Comment:
this should not return an interface unless it is a method factory.

Reply:
fixed